http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -226,10 +226,10 @@ object CommonTestData {
     proxyUrl = Runtime.getProxyUrl(proxyUrlBase, project, name1, Set(jupyterImage), Map.empty),
     status = RuntimeStatus.Unknown,
     labels = Map(),
-    jupyterExtensionUri = Some(GcsPath(GcsBucketName("bucketName"), GcsObjectName("extension"))),
-    jupyterUserScriptUri = Some(UserScriptPath.Gcs(GcsPath(GcsBucketName("bucketName"), GcsObjectName("userScript")))),
+    jupyterExtensionUri = Some(GcsPath(GcsBucketName("bucket-name"), GcsObjectName("extension"))),
```

Comment:
any specific reason for this change?

Reply:
Yeah, the route tests were encoding/decoding runtime requests and this was failing bucket name validation because capital letters aren't allowed:

https://github.com/broadinstitute/workbench-libs/blob/develop/model/src/main/scala/org/broadinstitute/dsde/workbench/model/google/GcsPathParser.scala#L15

I think `HttpRoutesSpec` was the first time we were decoding JSON -> model objects using the circe decoders.

Reply:
ah..that's a cool...